CODEOWNERS

```diff
@@ -17,4 +17,6 @@
 
 # By default the Android Components team will be the owner for everything in
 # the repo. Unless a later match takes precedence.
-*    @mozilla-mobile/act
+* @mozilla-mobile/act
+/automation @mozilla-mobile/releng
+/CODEOWNERS @mozilla-mobile/releng
```

Comment:
Can ACT own CODEOWNERS too?

Reply:
I think that if `ACT` owns all (`*`), ACT will own `CODEOWNERS` :)

Reply:
But this is going to be overridden by a later entry (at least for review requests). In this AC example ACT is not getting asked for review of glean code:
https://github.com/mozilla-mobile/android-components/blob/master/CODEOWNERS#L22-L24

Reply:
Oh, really, they don't stack?
Ahh, looking into the [docs](https://help.github.com/articles/about-code-owners/), the example explains that the latest match takes precedence, interesting!

pocmo you're on fire for catching these things, I really appreciate it :100: :fire: Thanks!